docs/writing-docs/tutorials.md

```diff
@@ -185,14 +212,15 @@ In each step, any text before the first code snippet or image is automatically d
 
 #### V2 Syntax
 
-If version 2 is specified in the tutorial metadata, the tutorial will only show hints that are explicitly defined with the hint tag, as follows: `<!-- HINT TEXT HERE -->`. One hint can be specified per step, and if no hint is specified all text (including blocks) will be displayed in the tutorial caption.
+If version 2 is specified in the tutorial metadata, the tutorial will only show hints that are explicitly defined with the hint tag, as follows: `#### ~ tutorialhint`. The hint automatically terminates at the next heading. One hint can be specified per step, and if no hint is specified all text (including blocks) will be displayed in the tutorial caption.
 
 ````markdown
-<!-- Try clicking on the 'Basic' drawer to find the blocks you need! Your code should look like this:
+#### ~ tutorialhint
+Try clicking on the 'Basic' drawer to find the blocks you need! Your code should look like this:
 
 ```blocks
 basic.showString("Micro!")
-``` -->
+```
```

Comment:
Don you have to close it with #### ~ ?

Reply:
No, I was going to update the docsrender to automatically infer the closing--Michal suggested any heading that is equal to or less than the opening, to allow hints with headings